tests/codegen/llvm/literal_strncmp.ll

```diff
@@ -11,6 +11,7 @@ target triple = "bpf-pc-linux"
 @AT_ = dso_local global %"struct map_t" zeroinitializer, section ".maps", !dbg !0
```

Comment:
Do we still need this codegen test at all now that string handling is all the same and string_equal_comparision.cpp also uses a literal?

Reply:
Sounds good to me.